### PR TITLE
refactor: update van_jsx

### PR DESCRIPTION
**Pull request by cqh963852**
relate issue https://github.com/vanjs-org/van/issues/233

- move jsx function to createElement file
- add setAttribute function
- use onClick like instead onclick while use JSX tag
- update children type to `VanNode`
- add OrVanAttribute, make tag can accept `(()=>T) | State<T>` type

@Tao-VanJS PTAL

### CODE REVIEW

File: addons/van_jsx/package.json [hunk 1]
```diff
@@ -2,12 +2,13 @@
   "name": "vanjs-jsx",
   "version": "0.1.0",
```
Comment (Tao-VanJS) [inline addons/van_jsx/package.json:3]: Should we upgrade the version?

**Review by Tao-VanJS — COMMENTED** [review]
Thanks @cqh963852

File: addons/van_jsx/package.json (hunk 1 above)
Comment (cqh963852) [reply addons/van_jsx/package.json:3]: Of course, let me upgrade it to 0.2.0

**Comment by Tao-VanJS** [comment]
Thanks a lot for the contribution!

**Pull request merged**